### PR TITLE
Add READMES to quark script showcases

### PR DESCRIPTION
**Pull request by zinwang**
# Description

This PR aims to update Quark Script to the repo and complete the README file of each Quark Script showcase.

# File Changes
 + Add CWE-327 Quark Script 
 + Add a README file to each showcase

### CODE REVIEW

File: CWE-94/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Detect CWE-94 in Android Application (ovaa.apk)
+
+This scenario seeks to find code injection in the APK file. See [CWE-94](https://cwe.mitre.org/data/definitions/94.html) for more details.
+
+Let’s use this [APK](https://github.com/oversecured/ovaa) and the above APIs to show how Quark script find this vulnerability.
+
+First, we design a detection rule loadExternalCode.json to spot on behavior uses method createPackageContext. Then, we find the caller method who calls the createPackageContext. Finally, we check if method checkSignatures is called in the caller method for verification.
+
+## Quark Scipt: CWE-94.py
```
Comment (haeter525) [inline CWE-94/README.md:9]: Please correct the spelling here.

File: CWE-312/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# Detect CWE-312 in Android Application (ovaa.apk)
+
+This scenario seeks to find cleartext storage of sensitive data in the APK file. See [CWE-312](https://cwe.mitre.org/data/definitions/312.html) for more details.
+
+Let’s use this [APK](https://github.com/oversecured/ovaa) and the above APIs to show how Quark script find this vulnerability.
+
+First, we designed a Frida script agent.js to hook the target method and get the arguments when the target method is called. Then we hook the method putString to catch its arguments. Finally, we use Ciphey to check if the arguments are encrypted.
```
Comment (haeter525) [inline CWE-312/README.md:7]: The links to the official websites of Frida and Ciphey is missing. Please correct it.

**Review by haeter525 — CHANGES_REQUESTED** [review]
Hi @zinwang. Thanks for the patch.
I've left one comment on the changes.

Also, I found some content is formatted differently from the document.
1. the APIs and the file name are not marked with backticks.
2. some words are not in bold.

Please fix them as well. Thanks.

**Comment by zinwang** [comment]
@haeter525 
I fixed grammar flaws in CWE-94 and added links to CWE-312.
Also, the formats in all the CWEs are the same as the document.
Please have a look! Thank you!

**Review by haeter525 — APPROVED** [review]
Thank @zinwang. LGTM!

**Pull request merged**